lib/test/CTestRunner.cc

```diff
@@ -54,6 +55,10 @@ const std::string CTestRunner::SKIP_FILE_NAME("unit_test_skip_dirs.csv");
 const std::string CTestRunner::XML_RESULT_FILE_NAME("cppunit_results.xml");
 
 CTestRunner::CTestRunner(int argc, const char** argv) {
+    ml::core::CLogger::instance().fatalErrorHandler([](std::string message) {
+        LOG_FATAL(<< message);
+        CPPUNIT_ASSERT(false);
```

Comment:
You could use `CPPUNIT_FAIL(message)`.  Then the message would be part of the test results reported through the framework, so would make it quicker to see what happened from a Jenkins test report for example.